doc/src/format.rs

```diff
@@ -0,0 +1,65 @@
+use itertools::Itertools;
+use solang_parser::{
+    doccomment::DocComment,
+    pt::{Base, FunctionDefinition, VariableDefinition},
+};
+
+use crate::output::DocOutput;
+
+pub trait DocFormat {
+    fn doc(&self) -> String;
+}
+
+impl<'a> DocFormat for DocOutput<'a> {
+    fn doc(&self) -> String {
+        match self {
+            Self::H1(val) => format!("# {val}"),
```

Comment:
is it possible to add pandoc attribute headers here too? 
[see pandoc docs about attr headers](https://pandoc.org/MANUAL.html#extension-header_attributes)

Also, idk if you may find this useful reference https://github.com/sambacha/go-natspec , would like to see forge doc as a separate binary if possible (if its a separate crate this should be the case correct?)